.vscode/extensions.json

```diff
@@ -6,8 +6,6 @@
     "recommendations": [
         "ms-vscode.cpptools",                   // C/C++ language support
         "twxs.cmake",                           // CMake language support
-        "vector-of-bool.cmake-tools",           // configure + build workflow for CMake-based projects
```

Comment:
Instead of removing completely, we should update the name - apparently the extension is now maintained by Microsoft and goes by the name `ms-vscode.cmake-tools`.

I also suggest just commenting out the second extension, until its maintainers produce a suitable update.